WakaTime/WakaTime.m

```diff
@@ -143,12 +143,25 @@ -(void)sendAction:(BOOL)isWrite {
         
         NSMutableArray *arguments = [NSMutableArray array];
         [arguments addObject:[NSHomeDirectory() stringByAppendingPathComponent:WAKATIME_CLI]];
+    
+        NSString* file = self.lastFile;
+        // Handle Playgrounds
+        if ([file.pathExtension isEqual: @"playground"]) {
+            [arguments addObject:@"--project"];
```

Comment:
Let's delegate project detection to [wakatime-cli](https://github.com/wakatime/wakatime) so the same project name is detected for playgrounds and other files in the same project.